deploy/microshift-bootc/config-svc/app.py

```
@@ -0,0 +1,1234 @@
+    if username != 'root':
+        return False
+    
+    try:
+        # Try using PAM authentication first
+        import pam
+        p = pam.pam()
+        return p.authenticate(username, password)
+    except ImportError:
+        # Fallback: use subprocess to authenticate via su
+        try:
+            result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)